### ISSUE TITLE
Signal display y-axis is inverted

### ISSUE THREAD

**Issue by rtoy** [hoch/canopy#9]
Run with the following simple demo:

```
var buffer = context.createBuffer(1, 10, context.sampleRate);
var s = context.createBufferSource();
var d = buffer.getChannelData(0);
for (var k = 0; k < d.length; ++k) {
  d[k] = 1;
}

s.buffer = buffer;
s.loop = true;

var g = context.createGain();

s.connect(g);
g.connect(context.destination);

g.gain.value = 0;
g.gain.setValueAtTime(1, 0);
g.gain.linearRampToValueAtTime(-1, 1.5);

s.start();
```

The curve starts at the bottom and goes up to the top.  But the code says the gain starts at 1 and linear ramps down to -1.  Thus the y axis is reversed, I think.

**Comment by hoch** [comment]
Hah! It was a stupid mistake. Will fix it soon.

**Comment by hoch** [comment]
Closed with bf4d3113d5272bba96d7209ee0a6e3230d607a4d.
